lib/token-types/bearer-token-type.js

```diff
@@ -0,0 +1,52 @@
+
+/**
+ * Module dependencies.
+ */
+
+var InvalidArgumentError = require('../errors/invalid-argument-error');
+
+/**
+ * Constructor.
+ */
+
+function BearerTokenType(accessToken, accessTokenLifetime, refreshToken, scope) {
+  if (!accessToken) {
+    throw new InvalidArgumentError('Missing parameter: `accessToken`');
+  }
+
+  this.accessToken = accessToken;
+  this.accessTokenLifetime = accessTokenLifetime;
+  this.refreshToken = refreshToken;
+  this.scope = scope;
```

Comment:
I'm not overly familiar with JWTs. Given that this module does not support them at this time, I'd prefer to leave this to a future PR if that's OK.